### PR TITLE
ncm-metaconfig: Singularity include v2.3 conf

### LINKED ISSUE #1130: ncm-metaconfig: Include Singularity 2.3.x  support
New Singularity version 2.3x has some new nice features to improve the security. The new values should be included into `singularity.conf`  file.

More info: https://singularity.lbl.gov/release-2-3

### PR DESCRIPTION
**Pull request by alvarosimon**
This PR is backwards compatible with previous Singularity versions. The new 2.3 `singularity.conf`  options are not taken into account by older versions.

* Fix #1130 : Include Singularity 2.3.x support

### CODE REVIEW

**Comment by stdweird** [comment]
retest this please

File: ncm-metaconfig/src/main/metaconfig/singularity/pan/schema.pan [hunk 1]
```diff
@@ -102,14 +102,50 @@ type singularity_container = {
     'dir' : absolute_file_path = '/var/singularity/mnt'
 } = dict();
 
+type singularity_sessiondir_max_size = {
+    @{This specifies how large the default sessiondir should be (in MB) and it will
+    only affect users who use the ""--contain" options and do not also specify a
```
Comment (stdweird) [inline ncm-metaconfig/src/main/metaconfig/singularity/pan/schema.pan]: `""` -> `"`

**Comment by stdweird** [comment]
@alvarosimon minor remark

**Comment by alvarosimon** [comment]
@stdweird done!

**Comment by stdweird** [comment]
@alvarosimon please modify the commit to reflect that this is ncm-metaconfig etc etc (eg same as the title of the PR)

**Comment by alvarosimon** [comment]
@stdweird done, I have changed the commit message

**Pull request merged**